public/style/components/_btn.scss

```diff
@@ -0,0 +1,48 @@
+.btn {
```

Comment:
I always err on the side of naming the scss file the same as the class so it's easier to find where the code is from looking at the finished product. Although button -> btn is a common abbreviation so shouldn't confuse too many people